src/Wasp.php

```diff
@@ -0,0 +1,239 @@
+			},
+			'stdin' => function() {
+				return new Input\StdIn();
+			},
+			'linter' => function() {
+				return new Linter\PhpLinter();
+			},
+		];
+	}
+
+	/**
+	 * Get global options that can apply to any command.
+	 * @return array InputOption and/or InputArgument objects
+	 */
+	protected function getGlobalOptions() {
+		return [
+			new InputOption('include', null, InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Additional files to include before executing.'),
+			new InputOption('disable-plugin', null, InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Plugins to disable when executing commands.'),
+			new InputOption('no-plugins', null, InputOption::VALUE_NONE, 'Disable all plugins when executing command.'),
+		];
+	}
+
+	/**
+	 * Get options that can be supplied for any command.
+	 * @return array option objects
+	 */
+	public function getDefaultGlobalInputDefinition() {
+		// merge application options with global options
+		return new PartialInputDefinition(array_merge($this->getApplication()->getDefinition()->getOptions(), $this->getGlobalOptions()));
+	}
+
+	/**
+	 * Get the global input definition object.
+	 * @return OptionalInputDefinition
+	 */
+	public function getGlobalInputDefinition() {
+		if (!$this->globalInputDefinition) {
+			$this->setGlobalInputDefinition($this->getDefaultGlobalInputDefinition());
+		}
+		return $this->globalInputDefinition;
+	}
+
+	/**
+	 * Set the global input definition object.
+	 * @param OptionalInputDefinition $inputDefinition
+	 */
+	public function setGlobalInputDefinition(InputDefinition $inputDefinition) {
+		$this->globalInputDefinition = $inputDefinition;
+	}
+
+	/**
+	 * Set the console application object.
+	 * @param Application $application application object
+	 */
+	public function setApplication(Application $application) {
+		$this->application = $application;
+	}
+
+	/**
+	 * Get the console application object.
+	 * @return Application application object
+	 */
+	public function getApplication() {
+		return $this->application;
+	}
+
+	/**
+	 * Parse lockfile and initialize any wasp plugins.
+	 * @param string $lockFile  contents of composer lock file
+	 * @param array  $disabled  slugs of plugins to disable
+	 */
+	protected function initializePluginsFromLock($lockFile, array $disabled = []) {
+		$logger = $this->getService('logger');
+		$lock = json_decode($lockFile, true);
+
+		if (!is_array($lock)) {
+			$logger->warning('Could not parse lock file');
+			return;
+		}
+
+		foreach (['packages', 'packages-dev'] as $key) {
+			if (empty($lock[$key]) || !is_array($lock[$key])) {
+				continue;
+			}
+
+			// check each package to see if it is a wasp plugin
+			foreach ($lock[$key] as $package) {
+				if (!isset($package['type']) || $package['type'] !== static::COMPOSER_TYPE) {
+					continue;
+				}
+
+				// disable plugin?
+				if (in_array($package['name'], $disabled, true)) {
+					$logger->debug("Plugin '{$package['name']}' is disabled");
```

Comment:
The other messages don't have quotes around `{$package['name']}`

Reply:
so it seems!